src/components/pages/Incidents/Map/Map.js

```diff
@@ -17,7 +17,9 @@ const Map = props => {
   const mapOptions = {
     fullscreenControl: false,
     styles: greystyle,
+    minZoom: 5,
```

Comment:
I know it was tough realizing how to work the min/max Zoom, I'm glad we got it working instead of abandoning this feature.